src/components/atoms/BaseCard/BaseCard.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState, KeyboardEvent } from 'react';
+import { createUseStyles } from 'react-jss';
+
+/**
+ * Properties for BaseCard component
+ */
+export interface BaseCardProps {
+  /** Element to show at card front face */
+  frontFace?: JSX.Element;
+  /** Element to show at card back face */
+  backFace?: JSX.Element;
+  /** Defines the card height */
+  height?: number;
+  /** Defines the card width */
+  width?: number;
+}
+
+const useStyles = createUseStyles({
+  // Generic classes
+  cardFace: {
+    backfaceVisibility: 'hidden',
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    color: 'cornSilk',
+    textAlign: 'center',
+    font: `3em/240px 'Helvetica Neue', Helvetica, sans-serif`,
+    boxShadow: '-5px 5px 5px #aaa',
+    transition: '0.6s',
+  },
```

Comment:
do we need all this styling if the card is an svg?

Reply:
Indeed all that part for font and boxShadow we can remove it, leaving only position only